nbs/core.ipynb

```diff
@@ -339,6 +339,8 @@
     "        self.id_col = id_col\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #100.            if "available_mask" not in temporal_cols:
</pre><p>The available_mask can be polars or pandas Series, I think it may be easier to define it as a bool array from the start, e.g.</p><pre class="ql-syntax" spellcheck="false">if "available_mask" in temporal_cols:
    available_mask = df["available_mask"].to_numpy().astype(bool)
else:
    available_mask = np.full(df.shape[0], True)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/894/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>